app/src/configurable_auth.h

```diff
@@ -27,6 +28,12 @@ namespace scitt
       auto identity = JwtAuthnPolicy::authenticate(tx, ctx, error_reason);
       if (!identity)
       {
+        CCF_APP_INFO(
+          "ClientRequestId={} Verb={} URL={} ErrorMessage={}",
```

Comment:
The error message is unstructured and will contain spaces, so it seems a bit odd to pretend it's parseable as key-value pair somehow. In the `generic_error_adapter` we log `... Code={code} {msg}` for internal errors and `... Code={code}` for user errors. Do you think we should stick to the same and emit a second log message like below?

`ClientRequestId={id} Code=InvalidAuthenticationInfo {msg}`

CCF uses `InvalidAuthenticationInfo` as code for 401 errors.

@plietar Any other ideas?

Reply:
>@plietar Any other ideas?

Only bad ones: Structure log messages as JSON.

Otherwise what you suggested with the parseable Code and trailing message is fine I think.